code/__DEFINES/components.dm

```diff
@@ -394,4 +408,4 @@
 #define COMSIG_XENO_MONKEY_CLICK_CTRL "xeno_monkey_click_ctrl"				//! from monkey CtrlClickOn(): (/mob)
 
 //Heretics stuff
-#define COMSIG_HUMAN_VOID_MASK_ACT "void_mask_act"
\ No newline at end of file
+#define COMSIG_HUMAN_VOID_MASK_ACT "void_mask_act"
```

Comment:
we don't have void path

Reply:
This signal is from the Mask of Madness port https://github.com/BeeStation/BeeStation-Hornet/pull/3446
The editor just automatically added a newline to the end of the file.